src/olympia/api/fields.py

```diff
@@ -548,3 +548,27 @@ def to_representation(self, value):
             if rep:
                 return rep
         return rep
+
+
+class LazyChoiceField(fields.ChoiceField):
```

Comment:
We should have dedicated tests for this

Reply:
https://github.com/mozilla/addons-server/pull/18428/commits/93d7e9848d1399e626c346931765fef3a1773a7a